docs/reference/index-modules/similarity.asciidoc

```diff
@@ -174,9 +174,15 @@ implementation used for these two methods, while not changing the
 `default`, it is possible to configure a similarity with the name
 `base`. This similarity will then be used for the two methods.
 
-You can change the default similarity for all fields by putting the following setting into `elasticsearch.yml`:
+You can change the default similarity for all fields by sending the
+following <<indices-update-settings,update settings>> request, note you
+must <<indices-open-close,close>> your index before sending the request
+and <<indices-open-close,open>> it again afterwards:
 
 [source,js]
 --------------------------------------------------
-index.similarity.default.type: classic
+PUT /my_index/_settings
+{
+  "index.similarity.default.type" : "classic"
+}
 --------------------------------------------------
```

Comment:
Could you give an example of setting it when creating the index and an example of updating it? I think that'd be more useful.